packages/babel-plugin-minify-mangle-names/README.md

```diff
@@ -71,3 +71,4 @@ require("babel-core").transform("code", {
 + `blacklist` - A plain JS Object with keys as identifier names and values indicating whether to exclude
 + `eval` - mangle identifiers in scopes accessible by eval
 + `keepFnName` - prevent mangler from altering function names. Useful for code depending on `fn.name`
++ `topLevel` - mangle topLevel Identifiers
```

Comment:
I'd still go with `topLevel` because it is simple and tells you what it does. I get the concerns that it doesn't tell you why toplevel bindings should/should not be treated as special cases. But I want to leave that to the user to decide. And here it's really upto the user to decide why he wants to use the option as it can be useful in multiple cases - whether he is parsing modules that he can choose to mangle toplevel, or whether his script is standalone that he can choose to mangle everything etc... In my view, the option is just to tell the user what it does rather than why you should do it.